### PR TITLE
Add GET operation to fishbowl, guest and user entities. Needed by api platform generator

### PR DESCRIPTION
**Pull request by aerrasti**
## Subject

Add GET operation to fishbowl, guest and user entities. Needed by api platform generator.

### CODE REVIEW

**Comment by aerrasti** [comment]
To generate interfaces files locally, just execute:
`NODE_TLS_REJECT_UNAUTHORIZED=0 npx @api-platform/client-generator --generator typescript https://localhost:8443 types/`

File: backend/src/Entity/Fishbowl.php [hunk 1]
```diff
@@ -40,7 +40,8 @@
  *     normalizationContext={"groups"={"fishbowl:read"}},
  *     denormalizationContext={"groups"={"fishbowl:write"}},
  *     collectionOperations={
- *         "post"={"security"="is_granted('ROLE_USER')"}
+ *         "get",
```
Comment (jordisala1991) [inline backend/src/Entity/Fishbowl.php:43]: You should take a look at this: https://api-platform.com/docs/core/operations/#enabling-and-disabling-operations

**Comment by jordisala1991** [comment]
> To generate interfaces files locally, just execute: `NODE_TLS_REJECT_UNAUTHORIZED=0 npx @api-platform/client-generator --generator typescript https://localhost:8443 types/`

Maybe you can also generate them, to ensure your changes are working good!

**Comment by aerrasti** [comment]
> > To generate interfaces files locally, just execute: `NODE_TLS_REJECT_UNAUTHORIZED=0 npx @api-platform/client-generator --generator typescript https://localhost:8443 types/`
> 
> Maybe you can also generate them, to ensure your changes are working good!

Thats why I leave it here! Im my old commit I generated them, but you told me to separate in two PR. Anyway, I can generate them :)

File: backend/src/Entity/Guest.php [hunk 2]
```diff
@@ -25,7 +26,14 @@
...
  *     normalizationContext={"groups"={"guest:read"}},
  *     denormalizationContext={"groups"={"guest:write"}},
- *     collectionOperations={"post"},
+ *     collectionOperations={
+ *         "get"={
+ *             "controller"=NotFoundAction::class,
+ *             "read"=false,
+ *             "output"=false,
+ *         },
+ *        "post"
```
Comment (jordisala1991) [inline backend/src/Entity/Guest.php:35]: missing space here

**Comment by aerrasti** [comment]
This PR is related with https://github.com/Stooa/Stooa/pull/66

**Review by eddsaura — APPROVED** [review]

**Pull request merged**